### PR TITLE
OHARA-1371 Update README.md

### PR DESCRIPTION
**Pull request by alan791205**

### CODE REVIEW

**Comment by g1geordie** [comment]
run swagger -l

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ gradle clean build -PskipManager
 ```
 gradle clean ohara-it:test -PskipManager -Pbrokers="" -Pworkers=""
 ```
-all integration tests currently live under the ohara-it. Without actual systems information, ohara-it use mini cluster 
+all integration tests are currently live under the ohara-it. Without actual systems information, ohara-it use mini cluster
```
Comment (konekoya) [inline README.md]: all integration tests are currently live under the ohara-it. Without actual system information, ohara-it will use a mini cluster

**Comment by chia7712** [comment]
Please file a jira first. see https://is-land.atlassian.net/secure/Dashboard.jspa

**Comment by chia7712** [comment]
@alan791205  the topic of pr should be same with jira.

**Comment by chia7712** [comment]
@alan791205  Please address @konekoya 's comment

File: README.md (hunk 1 above)
Comment (alan791205) [reply README.md]: Sure!

File: README.md [hunk 2]
```diff
@@ -71,6 +71,18 @@ see the source code of ohara-it for more details.
 gradle clean build -PskipManager
 ```
 
+
+### Run IT against actual systems
```
Comment (chia7712) [inline README.md]: Why you revert the deleted code?

Comment (alan791205) [reply README.md]: Done fixing, version confuse when fixing @konekoya 's request.

**Comment by vitojeng** [comment]
Thanks @alan791205 for the contribution!

**Pull request merged**